modules/rosa-nodes-machine-pools-local-zones.adoc

```diff
@@ -0,0 +1,74 @@
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-nodes-machinepools-configuring.adoc
+:_content-type: CONCEPT
+[id="rosa-nodes-machine-pools-local-zones_{context}"]
+= Local Zones
```

Comment:
This title doesn't seem like a procedure title to me.

```suggestion
= Configuring Local Zones for machine pools
```

Uninspired suggestion. :)

Reply:
I like your suggestion, but I'm going to keep the title as-is due to feedback from SMEs and PMs.

Reply:
I have made the suggested changes.